discv5/discv5-wire.md

```diff
@@ -318,7 +279,10 @@ REGCONFIRMATION notifies the recipient about a successful registration for the g
 topic. This call is sent by the advertisement medium after the time window for
 registration has elapsed on a topic queue.
 
-### TOPICQUERY Request (0x08)
+### TOPICQUERY Request (0x10)
```

Comment:
If this is hex, shouldn't it make more sense to `0x0a` ?

Reply:
Yes 🤦